cmd/registry/cmd/export/yaml_test.go

```diff
@@ -57,6 +58,8 @@ func TestExportYAML(t *testing.T) {
 		cmd := Command()
 		args := []string{"yaml", r}
 		cmd.SetArgs(args)
+		out := bytes.NewBuffer(make([]byte, 0))
+		cmd.SetOutput(out)
```

Comment:
FYI SetOutput is deprecated. The docs suggest using SetOut and SetErr. If both really do need silencing we can keep using this, but might have to fix the bug if we try to upgrade after they remove the function in a new version of cobra. I think I prefer two lines of code over someone having to deal with that eventually.

https://pkg.go.dev/github.com/spf13/cobra#Command.SetOutput

Reply:
Good to know - I was following [this](https://github.com/apigee/registry/blob/9bf1213bc7bf2ca5740bd12dd19a8fc50f7eeee0/cmd/registry/cmd/export/csv_test.go#L98), which also uses `SetOutput`. For clarity, I think it would be best to address these together with a PR when necessary.